test/features/proof_of_address.feature

```diff
@@ -0,0 +1,151 @@
+@browser
+Feature: Proof of address
+
+  Scenario: Test file upload for bank statement
+    Given I navigate to the SDK with PoA feature enabled
```

Comment:
The first few lines of every step are very similar, I would consider wrapping them into a step that takes the document type as an argument